lambdas/build-client-oauth-response/src/main/java/uk/gov/di/ipv/core/buildclientoauthresponse/BuildClientOauthResponseHandler.java

```diff
@@ -180,6 +182,20 @@ public Map<String, Object> handleRequest(JourneyRequest input, Context context)
                             auditEventUser,
                             new AuditRestrictedDeviceInformation(input.getDeviceInformation())));
 
+            var isReproveIdentity = clientOAuthSessionItem.getReproveIdentity();
+            if (Boolean.TRUE.equals(isReproveIdentity)
+                    && ipvSessionItem.getVot() != null
+                    && List.of(Vot.P0, Vot.P2).contains(ipvSessionItem.getVot())) {
```

Comment:
What's this check for? I'm not sure it's possible to reach here with a null VOT, but if they do, it's probably a failure.

Reply:
Mainly to make the existing tests pass - but I can change them instead :)